modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -107,11 +107,7 @@ func SetupICAPath(path *ibctesting.Path, owner string) error {
 		return err
 	}
 
-	if err := path.EndpointB.ChanOpenConfirm(); err != nil {
-		return err
-	}
-
-	return nil
+	return path.EndpointB.ChanOpenConfirm()
```

Comment:
so an update to the linter starting trigger all these?
sometimes I prefer an explicit function call, and there may be scenarios where we may want to log something. 

Happy to honour the linter here, just playing devils advocate. I think sometimes linters can be overly aggressive. We can always tweak the config later I guess.

Reply:
It might be beneficial to add `//nolint` directives in that case with an additional comment for context or is that too noisy? I can definitely remember cases where a linter was being silly.

Reply:
Yes, looks like golangci-lint was bumped to v1.52 and hell broke loose...

I had second thoughts when I was changing these as well, but thought the improvement was justified. I don't think the link would have complained if there was some extra logic inside the if statement.

Reply:
I agree with @damiannolan, I think it'd be nice to be able to turn off this specific linting check. I don't see a lot of value in enforcing it, but I do see value in not enforcing it